.rubocop.yml

```diff
@@ -1,11 +1,26 @@
+---
+
 inherit_from: .rubocop_todo.yml
 
 AllCops:
   TargetRubyVersion: 3.2
   NewCops: enable
 
-Style/FrozenStringLiteralComment:
-  Enabled: false
+Lint/ConstantDefinitionInBlock:
+  Enabled: true
+  Exclude:
+    - test/**/*.rb
+
+Metrics/BlockLength:
+  Max: 30
+  Exclude:
+    - test/**/*.rb
+
+Metrics/MethodLength:
+  Max: 11
```

Comment:
10 lines looks enough. Let's try to decompose large methods to keep this cop config default